### ISSUE TITLE
[a11y] 9.4.1.2 Name, role, value available

### ISSUE THREAD

**Issue by tschiebel** [owncloud/web#10739]
audit: https://infinite.owncloud.com/s/PnaAukHGGsLnrMM

https://bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-4-1-2-name-rolle-wert-verfuegbar

General notes:

- [x] The "Collapse/expand sidebar" button does not communicate its status to non-visual users. status to non-visual users. To improve this, the "aria-expanded" attribute should be used. should be used.
![image](https://github.com/owncloud/web/assets/132444655/f0a71025-c5fe-484a-b524-af03a94041b9)
Fixed via https://github.com/owncloud/web/pull/11386

### Personal

- [x] The use of "aria-expanded" attributes together with "span" elements
is not permitted. Further information on this can be found at
https://www.w3.org/TR/wai-aria-1.2/#aria-expanded. The attribute should be removed
be removed.
![image](https://github.com/owncloud/web/assets/132444655/b278d407-2915-4036-ad2f-f22923e05451)

- [x] #11375
is located in a "div" wrapper. It is not recommended to use the "aria-describedby" attribute together with
attribute together with "div" elements, as the output cannot be guaranteed by screen readers.
screen readers cannot be guaranteed.
![image](https://github.com/owncloud/web/assets/132444655/8e0db016-e01b-48c0-80f9-99d1bd311cd3)

- [x] In the tile view, the label of the sort button is provided by the sort value, for example
sorting value, for example "A-Z". The visual label
"Sort by" is not linked to the element, however, and using the "aria-describedby
the "aria-describedby" attribute does not help to understand the purpose of the button.
understand the purpose of the button. It is recommended to implement the element using the HTML "select" element
element and link it programmatically to the visual label,
which was realized by the "label" element.
![image](https://github.com/owncloud/web/assets/132444655/a2408b1c-f0aa-468d-80a0-38b9fb89af36)
- tbsbdr: problem is not understood, postponed.

- [x] In the mobile (responsive) view, the label of the menu button corresponds to the name of the current page. This could be confusing for some users
confusing for some users and it is not clear which page they are on.
Even if the menu is open, the active page (e.g. via the "aria-current" attribute) is not displayed.
attribute) is not displayed.
![image](https://github.com/owncloud/web/assets/132444655/c0911ba8-3c29-4ac4-b113-f49af7e9d0fd)
- tbsbdr: problem is not understood, postponed.

- [x] When focusing the "Files per page" combo box in the menu
"Show customization options for the file list" menu with the keyboard and a screen reader
activated screen reader, the element is automatically opened and the screen reader
screen reader focus is automatically set to the first element. This
screen reader users are not informed that this element is a combo box.
element is a combo box. A working example of this
problem can be found at
https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/comboboxselect-only/ can be found.
Also, the accessible name of the element "Files per page" is not
is not displayed.
![image](https://github.com/owncloud/web/assets/132444655/e8dbffe0-54d1-47b8-8234-c11aff7a76d7)
- tbsbdr: problem is not understood, postponed.

- [ ] See general notes.

### Shares

- [x] In the responsive (mobile) view, there is a problem with focusing with
with screen readers switched on: If the "Shared with me" menu is open,
when focusing on the "Shared with me" buttons, not only the label of the button is
label of the button, but also all links in the menu. This
is due to the use of the "aria-describedby" attribute. 
![image](https://github.com/owncloud/web/assets/132444655/5077b250-d898-4cac-b6cc-41f384ed72ee)
- tbsbdr: problem is not understood, postponed.
- @JammingBen : the way I understand it, this is not an issue anymore.

- [ ] See commentary on Personal.

### Spaces

- [x] In all modal dialogs such as "Change subtitle for this space 8A - Art", the "aria-labelledby" attribute is
the "aria-labelledby" attribute is placed on a "div" element instead of on the element with the
element with the role "dialog".
![image](https://github.com/owncloud/web/assets/132444655/8a2f07f7-2d4f-44a0-9a71-5ed19e29af01)

- [ ] See commentary on Personal.

### Deleted Files

- [ ] See general notes.

### Account

- [x] Modal dialog "Change password" - The "aria-labelledby" attribute is placed on a "div" element instead of on the element with the
element instead of the element with the role "dialog".
